diff --git a/lorem.rb b/lorem.rb
index 70714bc..f0253cf 100644
--- a/lorem.rb
+++ b/lorem.rb
@@ -1,9 +1,15 @@
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+# Lorem ipsum dolor sit amet, consectetur adipiscing elit.

  Sed laoreet rhoncus ligula a finibus.
-# In eu mi in mi semper tincidunt vel nec urna.
  Pellentesque nec viverra leo. Aenean rhoncus sapien at varius vulputate.

+Nunc urna orci, tincidunt rhoncus vulputate nec, malesuada eu nibh.
+# Aliquam quis est sit amet urna dapibus porta.
+# Etiam sit amet sollicitudin odio. In blandit porta.
+Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+# Suspendisse vitae blandit libero.
+Proin vulputate semper tellus, at imperdiet ligula mattis quis.
+
  Nam et dignissim ex.
-Integer volutpat, ante eu porttitor suscipit, felis erat pellentesque quam, sit amet efficitur libero magna porttitor purus.
+Integer volutpat, ante eu porttitor suscipit # , felis erat pellentesque quam, sit amet efficitur libero magna porttitor purus.
  Nulla leo libero, volutpat fringilla neque nec, bibendum placerat ex.
